swift_browser_ui_frontend/src/pages/BrowserPage.vue

```diff
@@ -1,45 +1,65 @@
 <template>
-  <div id="mainContainer">
-    <BrowserNavbar
-      :langs="langs"
-      :multiple-projects="multipleProjects"
-      :projects="projects"
-    />
-    <ProgressBar v-if="isUploading || isChunking" />
-    <b-breadcrumb
-      style="margin-left:5%;margin-top:1%;font-size:1rem;"
+  <div
+    id="mainContainer"
+    @dragenter="itemdrop = true"
+    @dragover="itemdrop = true"
+    @dragleave="itemdrop = false"
+  >
+    <label
+      v-if="itemdrop"
+      class="upload-draggable is-full-page"
+      style="width: 100%;height: 100%;margin: auto;"
+      @dragenter="dragHandler"
+      @dragover="dragHandler"
+      @dragleave="dragLeaveHandler"
+      @drop="navUpload"
     >
-      <b-breadcrumb-item
-        v-for="item in getRouteAsList ()"
-        :key="item.alias"
-        :to="item.address"
-        tag="router-link"
+      <p class="has-text-centered">
+        {{ $t('message.dropFiles') }}
+      </p>
+    </label>
+    <div v-else>
```

Comment:
this causes a rendering issue causing the `footer` to be moved up instead of on the lower part of the page:

what should be:
![image](https://user-images.githubusercontent.com/47524/147960861-6a3139e6-ba76-43ff-8320-6d5a4a984f96.png)

vs what is:
![image](https://user-images.githubusercontent.com/47524/147960908-2445008d-20cd-44a4-9597-98da4e44127a.png)

notice the position of the footer

Reply:
Fixed in commit df0169bd46f5b892a679736eba7832538af95a96